client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -528,62 +530,47 @@ where
 						self.sub_id,
 						err
 					);
-					let _ = sink.send(&FollowEvent::<String>::Stop);
-					return
+					return SubscriptionResponse::Event(FollowEvent::Stop)
 				},
 			};
 
 			for event in events {
-				let result = sink.send(&event);
-
-				// Migration note: the new version of jsonrpsee returns Result<(), DisconnectError>
-				// The logic from `Err(err)` should be moved when building the new
-				// `SubscriptionMessage`.
-
-				// For now, jsonrpsee returns:
-				// Ok(true): message sent
-				// Ok(false): client disconnected or subscription closed
-				// Err(err): serder serialization error of the event
-				if let Err(err) = result {
-					// Failed to submit event.
-					debug!(
-						target: LOG_TARGET,
-						"[follow][id={:?}] Failed to send event {:?}", self.sub_id, err
-					);
-
-					let _ = sink.send(&FollowEvent::<String>::Stop);
-					return
-				}
-
-				if let Ok(false) = result {
-					// Client disconnected or subscription was closed.
-					return
-				}
+				match sink
+					.send_timeout(sc_rpc::utils::to_sub_message(&event), Duration::from_secs(60))
```

Comment:
//cc @lexnv what to reckon to do here?

Reply:
This should be fine for chainHead use cases.
In the end, we want to backpressure connections that are not responsive to avoid wasting resources, at the same time 60 seconds should be plenty of time.
Further, the spec states that the node can submit the `Stop` event when it is overloaded, I interpret that to be the case here :D